packages/contracts-bedrock/src/libraries/Blueprint.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I would be very supportive of merging in the blueprint code first as its own PR to reduce the review size of this one

Reply:
Yea this PR got pretty big, let me do that split now

Reply:
Here it is: https://github.com/ethereum-optimism/optimism/pull/11700

Once merged, will update this branch so the blueprint diff goes away